### PR TITLE
[DONE] pms: wizard folio reservations

### PR DESCRIPTION
**Pull request by miguelpadin**

### CODE REVIEW

File: pms/wizards/wizard_folio.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                                        attrs="{'readonly':[('num_rooms_available','&lt;',1)]}"
+                                        force_save="1"
+                                    />
+                                    <field
+                                        name="price_per_room"
+                                        readonly="1"
+                                        force_save="1"
+                                    />
+                                    <!--
+                                        REVIEW:
```
Comment (miguelpadin) [inline pms/wizards/wizard_folio.xml]: @DarioLodeiros can you check this REVIEW, please? thx

File: pms/wizards/wizard_folio.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+                                },
+                            )
+                        )
+                    # remove old items
+                    old_lines = record.availability_results.mapped("id")
+                    for old_line in old_lines:
+                        cmds.append((2, old_line))
+
+                    record.availability_results = cmds
+
```
Comment (miguelpadin) [inline pms/wizards/wizard_folio.py:162]: @DarioLodeiros @eantones  Can you check this Review please? thx

File: pms/wizards/wizard_folio.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                            room_type_total_price_per_room += float(
+                                pricelist_item.price[2:]
+                            )
+                        else:
+                            # default price from room_type
+                            room_type_total_price_per_room += (
+                                room_type_iterator.product_id.list_price
+                            )
+
+                    # check there are rooms of the type
```
Comment (miguelpadin) [inline pms/wizards/wizard_folio.py:133]: @DarioLodeiros can you check this review please? thx

Comment (DarioLodeiros) [reply pms/wizards/wizard_folio.py:133]: I would show all, even if they have 0 availability, for UI issues

File: pms/wizards/wizard_folio.xml (hunk 1 above)
Comment (DarioLodeiros) [reply pms/wizards/wizard_folio.xml]: As we discussed, we will leave it like that until we have real feedback

File: pms/wizards/wizard_folio.py (hunk 3 above)
Comment (miguelpadin) [reply pms/wizards/wizard_folio.py:133]: Yes, it shows room types with no availability, that's correct. 
But I mean when exists one type room with no rooms setted as that type.
Captura de pantalla de 2021-01-05 15-32-56

Comment (DarioLodeiros) [reply pms/wizards/wizard_folio.py:133]: ah! ok, sorry. Its probably a configuration mistake, a think that is fine show this roomtype, but if you consider that no, leave it without showing

**Review by DarioLodeiros — APPROVED** [review]

**Pull request merged**